db/schema.rb

```diff
@@ -23,6 +23,9 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.bigint "user_id"
+    t.string "description"
```

Comment:
I need you to test for these new columns in the unit specs

Reply:
Done